README.md

```diff
@@ -97,6 +101,40 @@ module.exports = function (opts) {
 }
 ```
 
+### Stream concatenation / pipeline
+
+You can pipeline multiple transport as well.
```

Comment:
```suggestion
You can pipeline multiple transports:
```

It would be good to link to a concrete (real world) example in the `pino-syslog` docs when it is implemented.

Reply:
I Agree. I'll update the doc once this is implemented fully.